src/terminal/adapter/termDispatch.hpp

```diff
@@ -129,6 +129,7 @@ class Microsoft::Console::VirtualTerminal::TermDispatch : public Microsoft::Cons
     bool DoITerm2Action(const std::wstring_view /*string*/) override { return false; }
 
     bool DoFinalTermAction(const std::wstring_view /*string*/) override { return false; }
+    bool DoXtermJsAction(const std::wstring_view /*string*/) override { return false; }

```

Comment:
I'd call them VS Code sequences as they're not built into xterm.js or mentioned anywhere in the repo